apps/docs/content/components/ButtonGroup/index.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: ButtonGroup
+description: Use ButtonGroup to group related buttons together.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+![An image showing a ButtonGroup with a primary and a secondary button.](https://github.com/primer/brand/assets/912236/89ab4157-3c7a-4657-a7f0-1e481c943c41)
+
+## Usage
+
+Use ButtonGroup to group a set of actions that are related between them or to a single application. E,g, "Log in" and "Sign up" or "Try it out" and "Learn more".
+
+Use ButtonGroup to display a pair of primary and a secondary actions. The primary button is used for the main call to action while the secondary button is a related, less important action.
```

Comment:
Is there a preferred order of primary and secondary?

Can I render more than 2 buttons in a button group? All of the examples only have 2 buttons shown. If only 2 buttons are supported, maybe we should consider renaming this component "ButtonPair".